analysis/analysis_utils.py

```diff
@@ -309,16 +315,41 @@ def apply_mask(self, stack, mask_num_images=2, mask_threshold=120.):
                 The stack after the masks have been applied.
         """
         # mask pixels with any flags
+
+
+        # Hard coded mask functionality is deprecated. Use the new entries in
+        # the defaults dictionary
+
         #flags = ~0
-        # Only valid for LSST v20 difference images. Use with caution
-        mask_bits_dict = {'BAD': 0, 'CLIPPED': 9, 'CR': 3,
-            'DETECTED': 5, 'DETECTED_NEGATIVE': 6, 'EDGE': 4, 'INEXACT_PSF': 10,
-            'INTRP': 2, 'NOT_DEBLENDED': 11, 'NO_DATA': 8, 'REJECTED': 12,
-            'SAT': 1, 'SENSOR_EDGE': 13, 'SUSPECT': 7}
-        # Mask the following pixels
-        flag_keys = ['BAD', 'CR', 'INTRP', 'NO_DATA', 'SENSOR_EDGE', 'SAT',
-                    'SUSPECT', 'CLIPPED', 'REJECTED', 'DETECTED_NEGATIVE']
-        master_flag_keys = ['DETECTED']
+        # Only valid for LSST latest difference images. Use with caution
```

Comment:
Do we want to put this information somewhere else?

Reply:
Yes. In general, I think we should eventually start some formal (preferably mostly automatically generated) documentation for KBMOD.